### PR TITLE
adjust dynamic kademlia check

### PR DESCRIPTION
**Pull request by acud**
@janos @svetomir I think this is something we can start with and iterate on. Would be happy for you feedback.

ref https://github.com/ethersphere/bee/issues/1121

### CODE REVIEW

File: pkg/check/kademlia/dynamic.go [hunk 1]
```diff
@@ -102,53 +95,105 @@ func CheckDynamic(ctx context.Context, cluster *bee.Cluster, o Options) (err err
...
-
 		topologies, err := cluster.Topologies(ctx)
 		if err != nil {
 			return err
 		}
 
 		fmt.Println("Checking Kademlia")
 		if err := checkKademliaD(topologies); err != nil {
-			fmt.Printf("Kademlia check failed: %v\n", err)
+			fmt.Printf("Dynamic kademlia check failed: %v\n", err)
```
Comment (janos) [inline pkg/check/kademlia/dynamic.go:105]: Is this print needed as the error is retuned?

**Review by janos — APPROVED** [review]
LGTM

File: pkg/check/kademlia/dynamic.go (hunk 1 above)
Comment (acud) [reply pkg/check/kademlia/dynamic.go:105]: Will it be printed by cobra? I guess yes right?

**Review by svetomir — APPROVED** [review]

**Comment by svetomir** [comment]
@acud Looks good.

And, just a general question - should we remove fullConnectivity from static check also?
If answer is yes, maybe we could do it through this PR.

File: pkg/check/kademlia/dynamic.go (hunk 1 above)
Comment (janos) [reply pkg/check/kademlia/dynamic.go:105]: Yes, looks like that is the case, but actually, the exact caller is not relevant, it can be anything anytime.

Comment (acud) [reply pkg/check/kademlia/dynamic.go:105]: :+1:

**Review by janos — APPROVED** [review]

**Pull request merged**